src/composables/airtable.ts

```diff
@@ -53,7 +53,44 @@ export async function notifyGMNHUser(userWalletId: string, emailType: string, qu
       let userEmailAddress = output[0].get(emailAddressColumn)
       sendEmail(userEmailAddress, emailType, questionLink)
     }
+  }
+
+  export async function hasUserBeenAsked(userWalletId: string) {
+      /* Lookup if a GMNH user has been asked to enter their email. 
+      If not, ask them and mark them true so we don't ask them again
+            userWalletId --> user's wallet id
+            emailType --> type of email to send (tied to template)
+            questionLink --> /question/ link
     
+      */
+    
+        let hasBeenAskedToEnterEmail = 'hasBeenAskedToEnterEmail'
```

Comment:
maybe make this title case with "has" for consistency with rest of airtable columns?  a very minor thing